lib/PortingPhoneNumber.php

```diff
@@ -9,7 +9,18 @@
  */
 class PortingPhoneNumber extends ApiResource
 {
-    const OBJECT_NAME = "porting_phone_number";
+    const OBJECT_NAME = "number_order_phone_number";
```

Comment:
@weroh this was an error on our end and should remain as `porting_phone_number` the tests will fail until we fix the OpenAPI spec and Mock :(

Reply:
OK I have reverted this change in the latest update